### PR TITLE
refactor: add 'your alerts' to settings app

### PR DESCRIPTION
**Pull request by laurabeth**
The type of this PR is: Enhancement 

This PR addresses [GRO-654]

### Description

The wording in the ticket specifies that the code should be migrated, but this seems to have already been addressed. Here I added a tab to the settings app for easier navigation.

[GRO-654]: https://artsyproduct.atlassian.net/browse/GRO-654?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/v2/Apps/Settings/SettingsApp.tsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ const TABS = [
   { name: "Edit Settings", url: "/settings/edit-settings" },
   { name: "Collector Profile", url: "/settings/edit-profile" },
   { name: "Saves & Follows", url: "/settings/saves" },
+  { name: "Your Alerts", url: "/user/alerts" },
```
Comment (dzucconi) [inline src/v2/Apps/Settings/SettingsApp.tsx:16]: We should also change the route to `/settings/alerts` and add a redirect for the old route here: https://github.com/artsy/force/blob/a5a3defb3ba8ae6f382ffc5c3ebed85621d3f353/src/lib/middleware/hardcodedRedirects.ts#L77-L83

**Review by dzucconi — COMMENTED** [review]
I don't think this adds the tab bar at the top of the alerts page however; so when you tab over to it, it'll vanish.

File: src/v2/Apps/Settings/SettingsApp.tsx (hunk 1 above)
Comment (laurabeth) [reply src/v2/Apps/Settings/SettingsApp.tsx]: Got it! Thanks for pointing this out.

**Review by jacobherrington — APPROVED** [review]

**Review by damassi — APPROVED** [review]
Looks good! But do please [directly assign to someone](https://github.com/artsy/README/blob/main/playbooks/engineer-workflow.md#assignees-and-reviewers) before it's merged and update the title to match expected [RFC](https://github.com/artsy/README/issues/327) format.

**Comment by damassi** [comment]
Looks great 👍 , mergin'

**Pull request merged**